R/prophet.R

```diff
@@ -165,9 +165,18 @@ do_prophet_ <- function(df, time_col, value_col = NULL, periods, time_unit = "da
       else {
         time_unit_for_seq <- time_unit
       }
-      ts <- seq.POSIXt(as.POSIXct(min(aggregated_data$ds)), as.POSIXct(max(aggregated_data$ds)), by=time_unit_for_seq)
-      if (lubridate::is.Date(aggregated_data$ds)) {
-        ts <- as.Date(ts)
+      # Create periodical sequence of time to fill missing date/time
+      if (time_unit %in% c("hour", "minute", "second")) { # Use seq.POSIXt for unit smaller than day.
```

Comment:
Do you need the same processing in non-test mode?

Reply:
This is only for determining when to start test period. So no need of this for non-test mode.

Reply:
Added some more comment to explain what this is for.